tests/test_odgovlt.py

```diff
@@ -156,12 +164,12 @@ def test_OdgovltHarvester(app, db, mocker):
         '2014 m. vidutinio metinio paros eismo intensyvumo duomenys',
     ]
     clause = db.meta.tables['t_rinkmena'].select()
-    database_data = dict()
+    database_data = {}
     database_data_list = []
     for row in db.engine.execute(clause):
         for row_name in row.keys():
             database_data[row_name] = row[row_name]
```

Comment:
Why do you need that? Isn't `row` already is a dict? If this is a some kind of wrapper around dict, I think you can easily convert it to dict with `dict(row)`.

In other words, probably you can simply do:

```python
database_data_list.append(dict(row))
```